addons/stock/wizard/stock_track_confirmation.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields, tools
+
```

Comment:
missing new line

Reply:
it will be done.

Reply:
https://github.com/odoo/odoo/pull/21464#issuecomment-356009571